apps/web/src/contexts/auth/EthereumProviders.tsx

```diff
@@ -66,21 +67,25 @@ const myCustomTheme = merge(lightTheme(), {
   },
 });
 
-const { chains, provider } = configureChains(defaultChains, [
-  // TODO: move this to env and rotate creds
-  infuraProvider({ apiKey: '84842078b09946638c03157f83405213' }),
-  publicProvider(),
-]);
+const { chains, publicClient } = configureChains(
+  [mainnet, optimism, arbitrum, zora],
+  [
+    // TODO: rotate creds
```

Comment:
I moved the infura key to env like the TODO mentioned, but didn't rotate yet so left this part of the comment